src/emoji/EmojiPickerScreen.js

```diff
@@ -44,7 +44,7 @@ export default function EmojiPickerScreen(props: Props): Node {
   const { navigation, route } = props;
   const { onPressEmoji } = route.params;
 
-  const activeImageEmojiByName = useSelector(getActiveImageEmojiByName);
+  const activeImageEmojiObjects = useSelector(getActiveImageLocalEmoji);
```

Comment:
Let's find a name to consistently use for this. We generally have a convention that the result of selector `getFooBarBaz` gets passed around everywhere under the name `fooBarBaz`, and I've found that helpful for reducing the amount of explicit tracing around that one has to do to understand what some code is doing.

In the draft branch, I wrote the name "LocalEmoji" for a type that exists locally inside the `getFilteredEmojis` function, but I think "local" doesn't work as well when it's a type we're passing around widely.

Perhaps the type could be "EmojiForShared"? It's basically distinguished by being the form of an emoji object used in shared code.

And then the selector could be `getActiveImageEmojiForShared`… or perhaps better, simply `getActiveImageEmoji`. The particular type is there to be gotten from Flow, after all, when one needs to know it. And we fortunately don't have a counterpart in the other format that we need to give it a distinct name from.